src/sql/parts/grid/views/gridParentComponent.ts

```diff
@@ -236,6 +236,11 @@ export abstract class GridParentComponent {
 		}
 	}
 
+	protected isSelectAll(selectedRows: number): boolean {
```

Comment:
do we need to check the number of columns is also `totalColumns` or is that unimportant in this scenario?

Reply:
also, since we're doing `slickgrids.toArray()[0]`, is this going to work if there are numerous resultsets?